src/components/Form.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import styled from 'styled-components';
+
+
+const Formstyle = styled.form`
+  margin-top:1.8em;
+ input{
```

Comment:
This input is nested in this form as well as the button so nest them one level deep

Reply:
The input should get a
```css
border: 0;
padding: 10px;
```
to make it look more like the design